<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-A2S9SMV" failures="1" tests="2" name="StoreTest" time="14.181" errors="0" timestamp="2024-05-14T17:52:58 IST">
  <testcase classname="com.restassuredheroapp.testscripts.StoreTestCases" name="CreateOrder" time="11.533"/>
  <testcase classname="com.restassuredheroapp.testscripts.StoreTestCases" name="GetOrder" time="2.295">
    <failure type="java.lang.AssertionError" message="expected [111] but found [20886]">
      <![CDATA[java.lang.AssertionError: expected [111] but found [20886]
at com.restassuredheroapp.testscripts.StoreTestCases.GetOrder(StoreTestCases.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- GetOrder -->
</testsuite> <!-- StoreTest -->
